package.json

```diff
@@ -21,7 +21,7 @@
 	"scripts": {
 		"build": "babel --out-dir=dist source",
 		"dev": "babel --out-dir=dist --watch source",
-		"clean": "rm -rf dist",
+		"clean": "rmdir /s /q dist",
```

Comment:
I think replace this with something like this?

```
"clean": "npm run clean:linux"
"clean:linux": "rm -rf dist",
"clean:windows": "rmdir /s /q dist",
```

and then change the following:

```
"setup": "npm run setup:ubuntu",
"setup:ubuntu": "npm run clean && npm run build && npm i -g .",
```

into the following:

```
"setup": "npm run setup:linux",
"setup:linux": "npm run clean:linux && npm run build && npm i -g .",
"setup:windows": "npm run clean:windows && npm run build && npm i -g .",
```

You'll get the idea, so that we can work in both Windows and Linux. I'm using CodeSpaces. That's why I always use a Linux environment.

Reply:
@Abhinav-Bansal751 kindly check the following suggested changes.

Reply:
> I think replace this with something like this?
> 
> ```
> "clean": "npm run clean:linux"
> "clean:linux": "rm -rf dist",
> "clean:windows": "rmdir /s /q dist",
> ```
> 
> and then change the following:
> 
> ```
> "setup": "npm run setup:ubuntu",
> "setup:ubuntu": "npm run clean && npm run build && npm i -g .",
> ```
> 
> into the following:
> 
> ```
> "setup": "npm run setup:linux",
> "setup:linux": "npm run clean:linux && npm run build && npm i -g .",
> "setup:windows": "npm run clean:windows && npm run build && npm i -g .",
> ```
> 
> You'll get the idea, so that we can work in both Windows and Linux. I'm using CodeSpaces. That's why I always use a Linux environment.

can u check now,i have tried to make it work on both OS windows and ubuntu by using `||` in commands

Reply:
I leave a new comment from your latest changes.